js/files_test.ts

```diff
@@ -29,3 +29,84 @@ test(async function filesToAsyncIterator() {
 
   assertEqual(totalSize, 12);
 });
+
+testPerm({ write: true }, async function createFile() {
+  const tempDir = await deno.makeTempDir();
+  const filename = tempDir + "/test.txt";
+  let f = await deno.open(filename, deno.OpenMode.Write);
```

Comment:
Although I think it's nice to have the `OpenMode` enum, users are not going to be so explicit - they will use the short hand `"w"` instead. I think it would be better to use those in the tests.